pkg/tournament/tournament_classic_test.go

```diff
@@ -264,24 +270,24 @@ func TestClassicDivisionKingOfTheHill(t *testing.T) {
 
 	is := is.New(t)
 
-	roundControls := defaultRoundControls(rounds)
+	roundControls := defaultRoundControls(defaultRounds)
 
-	for i := 0; i < rounds; i++ {
+	for i := 0; i < defaultRounds; i++ {
 		roundControls[i].PairingMethod = realtime.PairingMethod_KING_OF_THE_HILL
 	}
 
-	tc, err := NewClassicDivision(playerStrings, playerRatings, roundControls, true)
+	tc, err := compactNewClassicDivision(defaultPlayers, roundControls, true)
 	is.NoErr(err)
 	is.True(tc != nil)
 
 	is.NoErr(validatePairings(tc, 0))
 
 	// Tournament should not be over
 
-	player1 := playerStrings[0]
-	player2 := playerStrings[1]
-	player3 := playerStrings[3]
-	player4 := playerStrings[2]
+	player1 := defaultPlayers.Persons[0].Id
+	player2 := defaultPlayers.Persons[1].Id
+	player3 := defaultPlayers.Persons[3].Id
+	player4 := defaultPlayers.Persons[2].Id
```

Comment:
(are player3, player4 being [3], [2] instead of [2], [3] intended?)

Reply:
This was a copy paste from another test where order used to matter but doesn't anymore due to changes in the way players are ranked initially. Which is player3 or player4 doesn't matter for this specific case, but it does look wrong and confusing, so I will switch it around to [2], [3] in the next push. Good catch.